apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -118,32 +118,24 @@ const CreateEventPage = ({ history }: CreateEventProps) => {
     }),
 
     onSubmit: async (values) => {
-      if (!values.endDate) values.endDate = cloneDeep(values.startDate);
-      addTimeToDate(values.startTime, values.startDate);
-      addTimeToDate(values.endTime, values.endDate);
+      notifyToast('Submitting Event...');
       try {
-        var uploadedFile = await uploadFile({ variables: { file: file } });
-        if (uploadedFile instanceof Error) {
-          throw uploadedFile;
-        }
+        const {
+          data: { uploadFile: uploadedFile },
+        } = await uploadFile({ variables: { file: file } });
+        await createEvent({
+          variables: {
+            createEventInput: {
+              ...payload,
+              image: { id: uploadedFile.id, mediaType: uploadedFile.mediaType },
+            },
+          },
+        });
+        updateToast(toast.TYPE.SUCCESS, 'Event Created');
```

Comment:
Like Omar mentioned, the user isn't redirected to home after creating the event since the `history.push` call was removed from `createMullEvent`, but not added back to `onSubmit`

```suggestion
        updateToast(toast.TYPE.SUCCESS, 'Event Created');
        history.push(ROUTES.HOME);
```